src/main/java/com/minecolonies/core/blocks/MinecoloniesFarmland.java

```diff
@@ -144,13 +150,22 @@ else if (i < 7)
         if (level.isRaining())
         {
             growthChance = 6;
-            BoneMealItem.addGrowthParticles(level, pos, 1);
         }
         if (aboveState.getBlock() instanceof MinecoloniesCropBlock cropBlock && rng.nextInt(100) <= growthChance)
         {
             cropBlock.attemptGrow(aboveState, level, pos.above());
+            Network.getNetwork().sendToPosition(new VanillaParticleMessage(pos.getX() + 0.5F, pos.getY() - 0.5F, pos.getZ() + 0.5F, ParticleTypes.HAPPY_VILLAGER),  new PacketDistributor.TargetPoint(pos.getX(), pos.getY(), pos.getZ(), BLOCK_BREAK_SOUND_RANGE, level.dimension()));
+        }
+    }
+
+    @Override
+    public void animateTick(final BlockState state, final Level level, final BlockPos pos, final RandomSource rng)
+    {
+        if (level.isRaining())
```

Comment:
doesnt this need an rng chance? else its probably too much

Reply:
I tested this, it seems animateTick is already offset decently. It had a similar effect as the random tick before.